docs/section4/4-3.md

```diff
@@ -0,0 +1,39 @@
+# Airside System – Rule 4-3
+
+**Rule ID:** 4-3
+ 
+**Rule Description:** Schedules for HVAC fans that provide outdoor air for ventilation shall run continuously whenever spaces are occupied.  
+**Rule Assertion:** P-RMR = expected value                                           
+**Appendix G Section:** Section G3.1-4 Schedules for the proposed building excluding exception #1. 
+**Appendix G Section Reference:** None
+**Data Lookup:** None 
+**Evaluation Context:** Evaluate each HeatingVentilationAirAconditioningSystem  
+
+**Applicability Checks:** 
+
+1. This rule does not apply when no heating and/or cooling system is to be installed, and a heating or cooling system is being simulated only to meet the requirements described in Section G3.1-10 HVAC Systems proposed column c and d.
+ 
+**Function Call:** 
+
+1. get_HVAC_systems_applicable_10cd
+
+**Manual Check:** None  
+
+**Rule Logic:**
+**Applicability Check 1:** 
+- Call function to get list of inapplicable HVAC systems: `inapplicable_HVAC_systems_list_p = get_HVAC_systems_applicable_10cd(U_RMR,P_RMR)`
+- For each HeatingVentilationAirAconditioningSystem in the P_RMR check if this check is applicable: `for each hvac_p in P_RMR...hvac_systems:`
+    - Check if hvac_p is not in the list of inapplicable systems, if it is not found then set applicability flag to true: `if hvac_p not in inapplicable_HVAC_systems_list_p: rule_applicability_flag = TRUE`   
+    - If rule_applicability_flag = TRUE for hvac_p: `if rule_applicability_flag == TRUE:`
+        - For each fan system in hvac_p: `fan_systems_p in hvac_p.fan_systems:`
```

Comment:
I submitted RMR issue recommending to require that a single fan_system is referenced by HeatingVentilationAirConditioningSystem data group. If this is accepted, line 28 will need to be updated to point to this system.

Reply:
Updated as this appears that it was accepted.